Video.js

```diff
@@ -571,10 +563,4 @@ Video.propTypes = {
   ...ViewPropTypes,
 };
 
-const RCTVideo = requireNativeComponent('RCTVideo', Video, {
```

Comment:
I had a quick review of requireNativeComponent api, It looks like 2 last props are not used...
Did you check on which react-native version it has been removed ?
I think this can be merged rapidly. I just merge your previous small Prs
Thanks you

Reply:
According to https://github.com/facebook/react-native/issues/28351#issuecomment-602755235, requireNativeComponent recieves only one argument since react-native@0.56